### PR TITLE
Fix boolean support with insert()

### LINKED ISSUE #83: INSERT statements fail with boolean arguments on PostgreSQL
```
Uncaught PDOException: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type boolean: ""
```

We need a variant of `buildInsertQuery()` that doesn't puke on boolean arguments.

### PR DESCRIPTION
**Pull request by paragonie-scott**
Fixes #83

### CODE REVIEW

File: src/EasyDB.php [hunk 1]
```diff
@@ -697,6 +696,44 @@ public function buildInsertQuery(string $table, array $columns): string
...
+        $columns = [];
+        $placeholders = [];
+        foreach ($map as $key => $value) {
+            if (\is_null($value)) {
+                $placeholders[] = 'NULL';
+            } elseif (\is_bool($value)) {
+                if ($this->dbEngine === 'sqlite') {
+                    $placeholders[] = $value ? '1' : '0';
+                } else {
+                    $placeholders[] = $value ? 'TRUE' : 'FALSE';
```
Comment (shadowhand) [inline src/EasyDB.php]: Maybe just depend on `latitude/latitude`? 😉 It is [consistent there](https://github.com/shadowhand/latitude/blob/3.1.0/src/Partial/Parameter.php#L19-L23).

Comment (paragonie-scott) [reply src/EasyDB.php]: That won't help for PHP 5 project

Comment (shadowhand) [reply src/EasyDB.php]: Maybe time to kill PHP 5 support then. :)

Comment (paragonie-scott) [reply src/EasyDB.php]: I wish. I'm proposing EasyDB to PHP 5.5 and 5.6-supporting projects to get them to stop being vulnerable to SQLi.

**Pull request merged**